modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Validation/ValidationDate.js

```diff
@@ -215,8 +214,13 @@ const CustomDates = ({
 									'quantity'
 								);
 							}}
+							onChange={(event) => {
+								const {value: newValue} = event.target;
+
+								setQuantity(newValue);
+							}}
```

Comment:
Taking advantage of [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) it can be converted to a single line:
```suggestion
							onChange={({target: {value}}) => setQuantity(value)}
```